README.md

```diff
@@ -208,6 +210,8 @@ prepare some data, and then fork out to several parallel tasks that use the disk
        height="213"/>
 </p>
 
+## GCE Persistent Disk
+
```

Comment:
The text contains a link to http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html already. Isn't that enough?